commands/scan/scan.go

```diff
@@ -454,6 +473,24 @@ func appendErrorSlice(scanErrors []formats.SimpleJsonError, errorsToAdd [][]form
 	return scanErrors
 }
 
+func cveListFromVulnerabilities(flatResults []services.ScanResponse) []string {
+	var cveList []string
+	var technologiesList []string
+	for _, result := range flatResults {
+		for _, vulnerability := range result.Vulnerabilities {
+			for _, cve := range vulnerability.Cves {
+				if !slices.Contains(cveList, cve.Id) && (cve.Id != "") {
+					cveList = append(cveList, cve.Id)
+				}
+			}
+			if !slices.Contains(technologiesList, vulnerability.Technology) && (vulnerability.Technology != "") {
+				technologiesList = append(technologiesList, vulnerability.Technology)
+			}
+		}
+	}
+	return cveList
```

Comment:
You can pass the ScanResponse directly to `RunApplicabilityScan` and it will caclulate the cve list for you. (I don't see you using the cveList anywehre else...)
No need for this method (it is also similar to `extractDependenciesCvesFromScan`)

Reply:
Done